src/ActionMenu/ActionMenu.tsx

```diff
@@ -9,6 +9,7 @@ import {Button, ButtonProps} from '../Button'
 import {useId} from '../hooks/useId'
 import {MandateProps} from '../utils/types'
 import {ForwardRefComponent as PolymorphicForwardRefComponent} from '../utils/polymorphic'
+import {Tooltip} from '../drafts/Tooltip/Tooltip'
```

Comment:
Wondering about backward compatibility, it's completely fair that ActionMenu will only behave nicely with the new tooltip. 

Does this have any impact on existing use cases where ActionMenu.Button is wrapped with old Tooltip? (not sure if folks already do that but maybe running integration tests on canary version will reveal them)

Reply:
>Does this have any impact on existing use cases where ActionMenu.Button is wrapped with old Tooltip? (not sure if folks already do that but maybe running integration tests on canary version will reveal them)

Great point! There is at least [one use case](https://github.com/github/github/blob/0669aabd5df584d2b68995fbcd4d65cfb5519c6c/app/assets/modules/hyperlist-web/components/list/header/CreateIssueButtonLoading.tsx#L21) I found and I replicated this in the storybook ([code](https://github.com/primer/react/pull/3394/commits/78a2f35819750986a4af7b41abc09bd779fe9aaa), [story](https://primer-dc05b46969-13348165.drafts.github.io/storybook/?path=/story/drafts-components-tooltip-features--test-old-tooltip-with-action-menu)) and seems like it is working nicely 🎉  I also pushed an [integration PR](https://github.com/github/github/pull/288842) but the CI checks aren't ran yet. I'll make sure it is all green there too ✅

Reply:
Note: I will revert that story after you view it - let me know if you think we should keep it though!

Reply:
Looks great!

Reply:
we chatted about them in a private channel but updating here for the records:
>Probably not. We can/should however write a test for it so that we can catch bugs in Tooltip v1 while it's still supported!

Great idea! Here is the commit : https://github.com/primer/react/pull/3394/commits/a05474e57f1f649958dad1c23517d9a18943b003

>Another side thought here: (because tooltips don't show up until hover/focus) Can we set a global variable on window or environment or even a secret prop that shows tooltip by default so that we can utilise visual regression testing?

When the stories are loaded [we added a line to press the tab key](https://github.com/primer/react/pull/3394/files#diff-6fbf2bea955d41c5b4aa83cfa4b3cc3b4d9682dda894da9a29e173211f020e83R18) to make sure the tooltips are open before taking the snapshots. With that, we utilise the VRT 🔥